ReactSkia/core_modules/RSkImageLoader.cpp

```diff
@@ -0,0 +1,137 @@
+/*
+* Copyright (C) 1994-2022 OpenTV, Inc. and Nagravision S.A.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+#include "include/core/SkImage.h"
+
+#include <cxxreact/JsArgumentHelpers.h>
+
+#include "ReactSkia/sdk/CurlNetworking.h"
+#include "ReactSkia/utils/RnsLog.h"
+#include "ReactSkia/components/RSkComponentImage.h"
+#include "RSkImageLoader.h"
+
+using namespace folly;
+
+namespace facebook {
+using namespace xplat;
+using namespace xplat::module;
+
+namespace react {
+
+RSkImageLoaderModule::RSkImageLoaderModule() {
+}
+RSkImageLoaderModule::~RSkImageLoaderModule() {
+  if(imageSizeMap_.size()!=0) {
+    ImageSizemap::iterator itr,it;
+    for(itr = imageSizeMap_.begin(); itr != imageSizeMap_.end(); ++itr) {
```

Comment:
Can we use range based loop? 
Instead of itr variable, need to add proper name for the variable based on its functionality